src/hooks/use-expiration-date.ts

```diff
@@ -5,12 +5,28 @@ import { useI18nData } from 'src/hooks/use-i18n-data';
 import { formatDistance } from 'src/lib/date';
 import { SupportedLocale } from 'src/lib/locale';
 
-function formatStringDate( ms: number, locale: SupportedLocale ): string {
-	const formatter = new Intl.DateTimeFormat( locale, {
-		day: 'numeric',
-		month: 'long',
-		year: 'numeric',
-	} );
+function formatStringDate(
+	ms: number,
+	locale: SupportedLocale,
+	format: 'long' | 'short' = 'short'
+): string {
+	const options: Intl.DateTimeFormatOptions =
+		format === 'short'
+			? {
+					day: 'numeric',
+					month: 'long',
+					year: 'numeric',
+			  }
+			: {
+					day: '2-digit',
+					month: 'short',
+					year: 'numeric',
+					hour: '2-digit',
+					minute: '2-digit',
+					second: '2-digit',
+					hour12: false,
+			  };
+	const formatter = new Intl.DateTimeFormat( locale, options );
```

Comment:
Could we define an object with the formats instead of using a ternary operator?

Something like:
```
type FormatKey = 'short' | 'long';
```

```suggestion
	const options: Record< FormatKey, Intl.DateTimeFormatOptions > = {
		short: {
			day: 'numeric',
			month: 'long',
			year: 'numeric',
		},
		long: {
			day: '2-digit',
			month: 'short',
			year: 'numeric',
			hour: '2-digit',
			minute: '2-digit',
			second: '2-digit',
			hour12: false,
		},
	};
	const formatter = new Intl.DateTimeFormat( locale, options[ format ] );
```

Reply:
That's a great idea! Thanks! I have made the change in https://github.com/Automattic/studio/pull/919/commits/e8711e3e4098243d55cbf4957fcca74fd9d40f58.